src/components/footer/footer.scss

```diff
@@ -15,21 +15,24 @@
   }
 }
 
+.App-footer {
+  background-color: #00478f;

```

Comment:
I think we need to talk to Design about this one. It's true that `StiriOficiale` has this color now, but in `Figma`, all the other projects have as primary color this one `#1C1080` 🤷‍♂

Reply:
Design confirmed that we should use `#00478f` for the footer to keep consistency with `StiriOficiale`. Buttons will still have `#1C1080` as the primary colour.